### PR TITLE
Master.users upd

### PR DESCRIPTION
**Pull request by alarmichanin**
Upd users part

### CODE REVIEW

File: users/admin.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ class UserAdmin(BaseUserAdmin):
     )
     readonly_fields = ("last_login",)
     # add_fieldsets is not a standard ModelAdmin attribute. UserAdmin
-    # overrides get_fieldsets to use this attribute when creating a user.
+    # overrides get_fieldsets to use this attribute when creating a user.
```
Comment (Pavliuk13) [inline users/admin.py:48]: God damn, what a fuck, man ? This may cause big bugs in our application!!

Comment (alarmichanin) [reply users/admin.py:48]: @Pavliuk13 bruh, whas ur name?

Comment (Pavliuk13) [reply users/admin.py:48]: @alarmichanin  My name is Alan. I'm chechenets

Comment (alarmichanin) [reply users/admin.py:48]: @Pavliuk13 fk u Alan

Comment (Pavliuk13) [reply users/admin.py:48]: @alarmichanin fuck you, gay!

Comment (Pavliuk13) [reply users/admin.py:48]: @KovalovV where are you ? I need your help here

File: users/tests.py [hunk 2]
```diff
@@ -38,13 +38,15 @@ def test_change_user_info(self):
...
 
+
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data.get("date_of_birth"), "2003-03-31")
         self.assertEqual(response.data.get("gender"), "male")
 
     def test_get_user_log(self):
         response = self.client.get(reverse("user_log"))
 
+
```
Comment (Pavliuk13) [inline users/tests.py:49]: It's a bullshit

**Review by Pavliuk13 — CHANGES_REQUESTED** [review]

File: users/admin.py (hunk 1 above)
Comment (alarmichanin) [reply users/admin.py:48]: @Pavliuk13 
u didn't understand 
https://youtube.com/shorts/Na7ErUfu7MQ?feature=share

Comment (Pavliuk13) [reply users/admin.py:48]: @alarmichanin I understood. You underestimate my memes knowledges

Comment (alarmichanin) [reply users/admin.py:48]: @Pavliuk13 mehhhh
you didn't

Comment (KovalovV) [reply users/admin.py:48]: guys guys
Biden bans our activities

Comment (Pavliuk13) [reply users/admin.py:48]: what a hell are you ?

Comment (KovalovV) [reply users/admin.py:48]: I'm from Yarik, I'll fuck you

File: users/tests.py (hunk 2 above)
Comment (alarmichanin) [reply users/tests.py:49]: Bulshit is ur Vitala

File: users/admin.py (hunk 1 above)
Comment (Pavliuk13) [reply users/admin.py:48]: You're not Yarik. And go fuck yourself

**Pull request merged**